lib/slax_web/websockets/issue.ex

```diff
@@ -29,6 +29,10 @@ defmodule SlaxWeb.Issue do
     end
   end
 
+  def scan_text(text) do
```

Comment:
Should the name `scan_text` speak a bit more towards why the text is being scanned or what the pattern is looking for?